test/unit/loops.spec.ts

```diff
@@ -763,21 +763,23 @@ test("forof forwarded lua iterator with tupleReturn", () => {
 });
 
 test.each([
-    "while (a < b) { i++; }",
-    "do { i++; } while (a < b)",
-    "for (let i = 0; i < 3; i++) {}",
-    "for (let a in b) {}",
-    "for (let a of b) {}",
+    "while (a < b) { i++; continue; }",
+    "do { i++; continue; } while (a < b)",
+    "for (let i = 0; i < 3; i++) { continue; }",
+    "for (let a in b) { continue; }",
+    "for (let a of b) { continue; }",
 ])("loop versions (%p)", loop => {
```

Comment:
Looking at what is being asserted here, this should probably have a better name, something like "loop continue for each lua target". Would probably also be nicer to have each version have its own test, but we can split that up in a later PR.